orchestration-shared/src/test/java/uk/gov/di/orchestration/shared/services/TokenValidationServiceTest.java

```diff
@@ -55,13 +55,6 @@ void shouldSuccessfullyValidateIDToken() {
         assertTrue(tokenValidationService.isTokenSignatureValid(signedIdToken.serialize()));
     }
 
-    @Test
-    void shouldNotFailSignatureValidationIfIdTokenHasExpired() {
-        Date expiryDate = NowHelper.nowMinus(2, ChronoUnit.MINUTES);
-        SignedJWT signedIdToken = createSignedIdToken(expiryDate);
-        assertTrue(tokenValidationService.isTokenSignatureValid(signedIdToken.serialize()));
-    }
-
```

Comment:
This test doesn't test the thing that has changed - it's testing something else, that we can use expired tokens. Instead of deleting the test, can we add an optional field to the token generator to take a value for the iat, and set it to something further in the past than the expiry date, so it still passes the new validation, but also tests what we're looking for?